app/jobs/fixity_check_job.rb

```diff
@@ -1,42 +1,63 @@
-class FixityCheckJob < Hyrax::ApplicationJob
```

Comment:
I meant to keep `< Hyrax::ApplicationJob`, if it's not there it came from a merge with master gone wrong.